clang/lib/Format/Format.cpp

```diff
@@ -2214,6 +2214,21 @@ FormatStyle::GetLanguageStyle(FormatStyle::LanguageKind Language) const {
 
 namespace {
 
+void replaceToken(const FormatToken &Token, FormatToken *Next,
+                  const SourceManager &SourceMgr, tooling::Replacements &Result,
+                  const char *Text = "") {
```

Comment:
```suggestion
                  StringRef Text = "") {
```

Reply:
I was going to use `StringRef` but instead settled on `const char *` because of #131299. I can switch to `StringRef` if you still think it's better.

Reply:
This is a local function and we use it with string literals. And if I looked correctly `tooling::Replacement` takes an `StringRef`.

Reply:
`StringRef` it is.